src/main/java/com/thealgorithms/maths/Area.java

```diff
@@ -61,83 +39,125 @@ private static double surfaceAreaCube(double sideLength) {
+        }
         return length * width;
     }
 
     /**
-     * Calculate surface area of a cylinder
+     * Calculate surface area of a cylinder.
      *
      * @param radius radius of the floor
      * @param height height of the cylinder.
      * @return volume of given cylinder
      */
-    private static double surfaceAreaCylinder(double radius, double height) {
+    public static double surfaceAreaCylinder(final double radius, final double height) {
+        if (radius <= 0) {
+            throw new IllegalArgumentException(POSITIVE_RADIUS);
+        }
+        if (height <= 0) {
+            throw new IllegalArgumentException(POSITIVE_RADIUS);
+        }
         return 2 * (Math.PI * radius * radius + Math.PI * radius * height);
     }
 
     /**
-     * Calculate the area of a square
+     * Calculate the area of a square.
      *
      * @param sideLength side length of square
      * @return area of given square
      */
-    private static double surfaceAreaSquare(double sideLength) {
+    public static double surfaceAreaSquare(final double sideLength) {
+        if (sideLength <= 0) {
+            throw new IllegalArgumentException("Must be a positive sideLength");
+        }
         return sideLength * sideLength;
     }
 
     /**
-     * Calculate the area of a triangle
+     * Calculate the area of a triangle.
      *
-     * @param base base of triangle
+     * @param base   base of triangle
      * @param height height of triangle
      * @return area of given triangle
      */
-    private static double surfaceAreaTriangle(double base, double height) {
+    public static double surfaceAreaTriangleRectangle(final double base, final double height) {
+        if (base <= 0) {
+            throw new IllegalArgumentException(POSITIVE_BASE);
+        }
+        if (height <= 0) {
+            throw new IllegalArgumentException(POSITIVE_HEIGHT);
+        }
         return base * height / 2;
     }
 
     /**
-     * Calculate the area of a parallelogram
+     * Calculate the area of a parallelogram.
      *
-     * @param base base of parallelogram
+     * @param base   base of parallelogram
      * @param height height of parallelogram
      * @return area of given parallelogram
      */
-    private static double surfaceAreaParallelogram(double base, double height) {
+    public static double surfaceAreaParallelogram(final double base, final double height) {
+        if (base <= 0) {
+            throw new IllegalArgumentException(POSITIVE_BASE);
+        }
+        if (height <= 0) {
+            throw new IllegalArgumentException(POSITIVE_HEIGHT);
+        }
         return base * height;
     }
 
     /**
-     * Calculate the area of a trapezium
+     * Calculate the area of a trapezium.
      *
-     * @param base1 upper base of trapezium
-     * @param base2 bottom base of trapezium
+     * @param base1  upper base of trapezium
+     * @param base2  bottom base of trapezium
```

Comment:
```suggestion
     * @param base1 upper base of trapezium
     * @param base2 bottom base of trapezium
```
No need for extra space

Reply:
Those whitespace are recommended by IntelliJ IDEA to put text on the same column. In fact, javadoc looks better

Reply:
Oh, thanks for sharing that. Everything looks greate